docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+
+- **known**: rate of new transactions arriving at the node that are ignored because the local node already knows about them.
+- **valid**: rate that node marks known transactions as valid
+- **invalid**: rate that node marks known transactions as invalid
+- **underpriced**: rate that node marks transactions paying insufficient gas as invalid
+- **executable discard**: rate that valid transactions are dropped from the transaction pool, e.g. because it is already known.
+- **executable replace**: rate that valid transactions are replaced with a new one from same sender with same nonce but higher gas
+- **executable ratelimit**: rate that valid transactions are dropped due to rate-limiting
+- **executable nofunds**: rate that valid transations are dropped due to running out of ETH to pay gas
+- **gapped discard**: rate that queued transactions are discarded from the transaction pool
+- **gapped replace**: rate that queued transactions are replaced with a new one from same sender with same nonce but higher gas
+- **gapped ratelimit**: rate that queued transactions are dropped due to rate limiting
+- **gapped nofunds**: rate that queued transactions are dropped due to running out of ETH to pay gas
+
+![The tx processing panel](/images/docs/grafana/tx-processing.png)
+
+#### Block propagation
+
+Block propagation metrics track the rate that the local node hears about, receives and broadcasts blocks. This includes:
+
+- **ingress announcements**: counts the number of inbound announcements processed by the local node. Announcements are messages from peers that signal that they have a block to share.
+- **known announcements**: counts the announcements that are ignored because the local node is already aware of them.
+- **malicious announcements**: counts the number of announcements from peers that are determined to be malicious, e.g. because they are trying to mount a denial-of-service attack on the local node.
+- **ingress broadcasts**: Blocks downloaded from peers
+- **known broadcasts**: counts all blocks that have been broadcast by peers including those that are too far behind the head to be downloaded.
+- **malicious broadcasts**: counts all blocks that are determined to be malicious
+
+
+#### Transaction propagation
+
+Transaction propagation tracks the sending and receiving of transactions on the peer-to-peer network. This includes:
+
+- **ingress announcements**: counts inbound announcements (notifications of a transaction's availability)
+- **known announcements**: counts announcements that are ignored because the local node is already aware of them
+- **underpriced announcements**: counts all announcements that never get fetched because they pay too little gas 
+- **malicious announcements**: counts announcements that are dropped because they appear malicious 
+- **ingress broadcasts**: counts all transactions fetched from peers
```

Comment:
the number of transactions propagated from peers per second

Reply:
```suggestion
- **ingress broadcasts**: the number of transactions propagated from peers per second
```